lambdas/service/app.py

```diff
@@ -824,6 +851,58 @@ def self_url(endpoint_path=None):
     return retry_url
 
 
+def file_url(uuid, **params):
```

Comment:
Inconsequential change. Just changes the order of two methods. Remove.

Reply:
It was a rebasing issue. I moved it back.